oak-parent/pom.xml

```diff
@@ -718,6 +718,13 @@
         <version>${testcontainers.version}</version>
         <scope>test</scope>
       </dependency>
+      <!-- Testcontainers dependency -->
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>testcontainers</artifactId>
+        <version>${testcontainers.version}</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
same here

Reply:
removed in https://github.com/apache/jackrabbit-oak/pull/1459